extensions/amp-story-shopping/amp-story-shopping.md

```diff
@@ -132,6 +132,10 @@ Inline data may be served from cache which may take time to propogate. `src` JSO
 }
 ```
 
+### Product JSON Validation
```

Comment:
```suggestion
### Product JSON Schema
```

wdyt about using "schema" in the title so terminology us visible in the table of contents?